src/roster40k.ts

```diff
@@ -918,23 +918,22 @@ function ParseModelProfiles(profiles: Element[], model: Model, unit: Unit) {
             const explosion = ParseExplosionProfile(profile);
             model._explosions.push(explosion);
         } else if (typeName == "Psyker") {
-            const psyker = ParsePsykerProfile(profile);
-            model._psyker = psyker;
+            model._psyker = ParsePsykerProfile(profile);
         } else {
             // Everything else, like Prayers and Warlord Traits. 
             if (!unit._abilities[typeName]) unit._abilities[typeName] = new Map();
-            ParseProfileCharacteristics(profile, profileName, unit._abilities[typeName]);
+            ParseProfileCharacteristics(profile, profileName, typeName, unit._abilities[typeName]);
         }
     }
 }
 
-function ParseProfileCharacteristics(profile: Element, profileName: string, map: Map<string, string | null>) {
+function ParseProfileCharacteristics(profile: Element, profileName: string, typeName:string,  map: Map<string, string | null>) {
     const chars = profile.querySelectorAll("characteristics>characteristic");
     for (const char of chars) {
         if (!char.textContent) continue;
 
         const charName = char.getAttribute("name");
-        if ((charName === "Description") || (charName === "Ability") || (charName === "Effect") || (charName === "Bonus") || (charName === 'Capacity')) {
+        if ((charName === "Description") || (charName === "Ability") || (charName === "Effect") || (charName === "Bonus") || (charName === 'Capacity') || typeName === 'Invocations') {
```

Comment:
Please add a test case for this to prevent regressions in the future — you can put a small .rosz file in the test/ directory, and then run write40kRosterExpectation.ts to write the test file (instructions to run are in that file)